source/content/wordpress-best-practices.md

```diff
@@ -51,3 +51,44 @@ This article provides suggestions, tips, and best practices for developing and m
 * Verify [Global CDN caching](/test-global-cdn-caching/) is working on your site.
 
 * Follow our [Frontend Performance](/guides/frontend-performance/) guide to tune your WordPress site.
+
+## Avoid XML-RPC Attacks
+The `/xml-rpc.php` path is a potential security risk for WordPress sites. It can be used by bad actors to brute force administrative usernames and passwords. All calls made to this file must supply a plain text username and password. If a valid combination is found, a success message is returned letting the bad actor know they can log in via the standard `/wp-login.php` method.
+
+This can be surfaced by reviewing your site's `nginx-access.log` for the Live environment. If you leverage [GoAccess](/nginx-access-log/), you might see something similar to the following:
+
+```none
+2 - Top requests (URLs)                                  Total: 366/254431
+
+Hits Vis.     %   Bandwidth Avg. T.S. Cum. T.S. Max. T.S. Data
+---- ---- ----- ----------- --------- --------- --------- ----
+2026   48 0.77%   34.15 KiB   1.27  s  42.74 mn  38.01  s /xmlrpc.php
+566   225 0.21%   12.81 MiB   4.08  s  38.45 mn  59.61  s /
+262    79 0.10%  993.71 KiB   2.32  s  10.14 mn  59.03  s /wp-login.php
+```
+
+Pantheon recommends disabling XML-RPC, given the WordPress Rest API is a stronger and more secure method for interacting with WordPress via some external service.
+
+### Disable XML-RPC via Pantheon.yml
+
+Add the following configuration to your [`pantheon.yml`](/pantheon-yml/) file:
+
+  ```yml
+  protected_web_paths:
+    - /xmlrpc.php
+  ```
+
+This method is more performant than disabling via PHP since this won't involve bootstrapping the application.
+
+### Disable XML-RPC via PHP
+
+This method has the advantage of being toggleable without deploying code, by activating or deactivating a custom MU Plugin.
+
+1. If you have not already created an **MU Plugin** (Must Use Plugin) to maintain custom code, review [Create a WordPress MU Plugin for Actions and Filters](/mu-plugin/).
```

Comment:
@alexfornuto MU Plugins can't be disabled, so as is the advantage of toggling does not apply

Suggest we advise they use this WP-CLI command instead (align with @sarahg's feedback)
https://developer.wordpress.org/cli/commands/scaffold/plugin/

Reply:
Would not recommend a MU plugin for this, just a regular one.

Reply:
@sarahg @rachelwhitton I updated this section as suggested, but I know goose-egg about making custom plugins, and don't know if this needs any other information besides the what-do code to disable XML.

Reply:
@alexfornuto you can assign back to me and I'll push a commit for review tomorrow